### DISCUSSION TITLE
How to configure a TypeScript script package to install @acme/db

### DISCUSSION THREAD

**Discussion by hayata-suenaga** [t3-oss/create-t3-turbo#1113] [Q&A]
I added a package that has one file `index.ts` file that imports from `@eisuke/db/client`. The script is supposed to be run locally to populate database with some data.

I'm trying to figure out the best way to execute the script. I tried to use `ts-node`. However, `ts-node` seems to be having difficulty to handle the import statement from the internal pacake.

What is the best way to create a simple TypeScript script that relies on internal packages?

**Comment by juliusmarminge** [comment]
Doesnt `node --import tsx script.ts` work?
